source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -77,14 +78,45 @@ TextureBaker::TextureBaker(unsigned int width, unsigned int height, Image::BaseT
 #endif
         _colorSpace = SRGB_TEXTURE;
     }
-    else
-    {
+	else
```

Comment:
Accidental whitespace changes here?

Reply:
Can you provide a automated clang format like formatter?

Reply:
I think that's a great suggestion for the future, as noted in #652, but so far our contributors have followed a "when in Rome" policy with respect to coding style.